BundesIdent/Helpers/ABTester.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
+import Analytics
+import UnleashProxyClientSwift
+
+enum ABTest: String, CaseIterable {
+    case none
+}
+
+final class Unleash: ABTester {
+
+    init(url: String, clientKey: String, analytics: AnalyticsClient, issueTracker: IssueTracker) {
+        unleash = .init(unleashUrl: url, clientKey: clientKey, refreshInterval: .max, appName: "bundesIdent.iOS")
+        unleash.context["supportedToggles"] = ABTest.allCases.map(\.rawValue).joined(separator: ",")
```

Comment:
No, it’s not because of unleash. It’s because our `enum` needs at least one case to conform to `RawRepresentable`. I wanted to keep the enum eve when we are not running any tests. Sending `"none"` to Unleash doesn’t break anything, but we could also exclude it here. The idea is to come up with the system that will require the least amount of changes when a feature flag introduced or deprecated. Do you have another idea?